jar-infer/jar-infer-cli/build.gradle

```diff
@@ -39,12 +40,53 @@ shadowJar {
 shadowJar.dependsOn jar
 assemble.dependsOn shadowJar
 
-apply plugin: 'com.vanniktech.maven.publish'
+tasks.withType(PublishToMavenRepository) {
```

Comment:
Can we add a comment here?  Indicating we are disabling publication of the "standard" jar artifact